Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -603,6 +605,35 @@ private extension ProductStore {
             completion(result)
         }
     }
+
+    func generateProductDetails(siteID: Int64, scannedTexts: [String], completion: @escaping (Result<ProductDetailsFromScannedTexts, Error>) -> Void) {
+        let prompt = [
+            "Write a name and description of a product for an online store given the array of scanned text strings from a packaging photo at the end.",
+            "Return only a JSON with the name in `name` field, description in `description` field, " +
+            "and the detected language as the locale identifier in `language` field.",
+            "The output should be in valid JSON format.",
+            "Identify the language detected in the array and use the same language to write the name and description.",
```

Comment:
Nit: should we simplify this as `Detect the language in the array...`

Reply:
Sure, updated in https://github.com/woocommerce/woocommerce-ios/pull/10183/commits/e2be484e443f0cc36aa0d1183344633d06b33411. I'll test it more after the PR is merged.